### PR TITLE
Daniel submission

### PR DESCRIPTION
**Pull request by dannyozh**
Deliverable Submission

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5): 4

Completeness Level (1-5): 4

What did you think of this deliverable?:
Have a better idea of how react, express and SQL all come together, although it gives me a headache trying to comprehend the 'path' of it all.

### CODE REVIEW

**Comment by dannyozh** [comment]
Deliverable Submission (part 2)

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5): 4

Completeness Level (1-5): 4

What did you think of this deliverable?:
Took me 5 hours. Achieved MVP. Also, understood what .map does, i think. HOMG.....

File: index.js [hunk 1]
```diff
@@ -46,100 +46,282 @@ app.engine('jsx', reactEngine);
...
+			res.send(result.rows);
+		}
+	});
+});
+
+// see which songs are in which playlists
+app.get('/playlists/:id/songtitles', (req, res) => {
+	console.log('getting you all song names in this playlist');
+	let playlistId = parseInt(req.params.id);
+	let queryString = `SELECT songs.title, songs.album FROM songs INNER JOIN playlists_songs ON (playlists_songs.songs_id = songs.id) WHERE (playlists_songs.playlists_id = ${playlistId})`;
```
Comment (kach92) [inline index.js]: avoid using js string interpolation, use $1,$2 instead

Comment (dannyozh) [reply index.js]: Noted!

**Comment by dannyozh** [comment]
Deliverable Submission (part 2)

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5): 4

Completeness Level (1-5): 3

What did you think of this deliverable?:
Cookies are cool. Am curious to know if there's a way to erase them permanently?

**Comment by kach92** [comment]
> Deliverable Submission (part 3)
> 
> Please describe your comfort and completeness levels before submitting.
> 
> Comfort Level (1-5): 4
> 
> Completeness Level (1-5): 3
> 
> What did you think of this deliverable?:
> Cookies are cool. Am curious to know if there's a way to erase them permanently?

there is

**Pull request closed without merging**